backend/src/main/java/com/benine/backend/http/CameraInfoHandler.java

```diff
@@ -29,18 +31,15 @@ public CameraInfoHandler(CameraController controller) {
    * @throws IOException When writing the response fails.
    */
   public void handle(HttpExchange exchange) throws IOException {
-    //TODO add logging stuff
+    getLogger().log("Got an http request with uri: " + exchange.getRequestURI(), LogEvent.Type.INFO);
     JSONObject jsonObj = new JSONObject();
     ArrayList<Camera> cameras = getCameraController().getCameras();
     JSONArray camerasJSON = new JSONArray();
     for (Camera cam : cameras) {
```

Comment:
Maybe the camerasJSON object can be returned by a separate method, to split handling from generating the JSON a bit more.